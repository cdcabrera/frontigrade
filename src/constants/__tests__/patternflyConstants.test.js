import bgImages from '../patternflyConstants';

describe('PatternflyConstants', () => {
  it('Should have specific bg images', () => {
    expect(bgImages).toMatchSnapshot('background image strings');
  });
});
