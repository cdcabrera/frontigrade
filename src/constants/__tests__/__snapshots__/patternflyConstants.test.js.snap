// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PatternflyConstants Should have specific bg images: background image strings 1`] = `
Object {
  "filter": "background-filter.svg#image_overlay",
  "lg": "pfbg_1200.jpg",
  "sm": "pfbg_768.jpg",
  "sm2x": "pfbg_768@2x.jpg",
  "undefined": "pfbg_2000.jpg",
  "xs": "pfbg_576.jpg",
  "xs2x": "pfbg_576@2x.jpg",
}
`;
